### DISCUSSION TITLE
Limiting the voice's radio based on distance

### DISCUSSION THREAD

**Discussion by ManuelAGC** [mozilla/hubs#3432] [General Questions & Discussion] [answered]
Hi, I was wondering if anybody knows how to limit the radio in which you can hear someone talking. I am worried that in a scene with 20 people you are able to hear all of them at the same time despite putting distance between yourself and them. I have tried with a friend and we could still hear each other at a fairly long distance away. Does anybody know how to fix this?

**Comment by rawnsley** [comment, marked as answer]
Each scene had audio settings that can influence this. Check the root scene node in Spoke.

**Reply by ManuelAGC** [reply]
Thanks a lot! That is exactly what I was looking for.
